lib/nixd/src/Diagnostic.cpp

```diff
@@ -2,6 +2,30 @@
 
 #include "lspserver/Protocol.h"
 
+using std::string;
```

Comment:
remove this statement, and use qualified id `std::string`. Because `string` is a common word in naming convention.